coral/index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.png" />
```

Comment:
Did you copy the asset from the Springboot assets? I think we should use the proper image type as the HTML attribute:
`<link rel="icon" type="image/png" href="/favicon.png" />`